edx_exams/apps/api/v1/views.py

```diff
@@ -384,11 +387,22 @@ def get(self, request, exam_id):
 class ExamAttemptView(ExamsAPIView):
     """
     Endpoint for the ExamAttempt
+
+    Given the id of the attmpt, this view will
     /exams/attempt/<attempt_id>
 
     Supports:
+        HTTP GET: Get an attempt's status.
         HTTP PUT: Update an attempt's status.
 
+    HTTP GET
+    **Returns**
+    ExamAttempt = {
```

Comment:
NIT: This should be formatted like a dict, and look similar to https://github.com/edx/edx-exams/blob/870a684a342b64266f09de5df3cd89ef316f58dc/edx_exams/apps/api/v1/views.py#L206-L208